org.libretro.RetroArch.json

```diff
@@ -23,7 +23,84 @@
     "shared-modules/SDL/SDL_mixer-1.2.12.json",
     "shared-modules/SDL/SDL_net-1.2.8.json",
     "shared-modules/SDL/SDL_ttf-2.0.11.json",
-    "shared-modules/udev/udev-175.json",
+    {
+	  "name": "eudev",
+	  "rm-configure": true,
+	  "config-opts": [
+		"--disable-hwdb",
+		"--disable-logging",
+		"--disable-introspection",
+		"--disable-keymap",
+		"--disable-mtd_probe",
+		"--disable-programs"
+	  ],
+	  "cleanup": [
+		"/include",
+		"/etc",
+		"/libexec",
+		"/sbin",
+		"/lib/pkgconfig",
+		"/man",
+		"/share/aclocal",
+		"/share/doc",
+		"/share/gtk-doc",
+		"/share/man",
+		"/share/pkgconfig",
+		"*.la",
+		"*.a"
+	  ],
+	  "sources": [
+		{
+		  "type": "archive",
+		  "url": "https://dev.gentoo.org/~blueness/eudev/eudev-3.2.7.tar.gz",
+		  "sha256": "3004614bd253c1f98558460215027aaf60d7592c70be27fd384ec01db87bf062"
+		},
+		{
+		  "type": "script",
+		  "dest-filename": "autogen.sh",
+		  "commands": [
+		    "autoreconf -vfis"
+		  ]
+		}
+	  ]
+	},
+    {
+	  "name": "libusb",
```

Comment:
I'm confused why this module has so much extra stuff compared to the dolphin one: https://github.com/flathub/org.DolphinEmu.dolphin-emu/blob/master/org.DolphinEmu.dolphin-emu.json#L26

Reply:
It's because the Dolphin core is linked to libusb and a probably the systemd version of libudev which has a different soname, eudev is derived from the systemd libudev and has the same ABI and soname.

The other possible solution is to change how the Dolphin core that RetroArch downloads is built to not link to either libudev or libusb.

Reply:
> The other possible solution is to change how the Dolphin core that RetroArch downloads is built to not link to either libudev or libusb.

Would certainly love updates in the Dolphin core https://github.com/libretro/dolphin .

The following seems to link them:
https://github.com/libretro/dolphin/blob/master/CMakeLists.txt#L634-L648

We could bring the libusb source directly in.

Reply:
> probably the systemd version of libudev which has a different soname, eudev is derived from the systemd libudev and has the same ABI and soname.

Makes sense :+1:

Reply:
but you can still simplify the `libusb` module a bit.